README.md

```diff
@@ -60,7 +60,7 @@ Web version is also available now. but only transmitter version. [Web version.]
 ## Sample Projects
 - [Euphony Listener](https://github.com/designe/euphony-listener) : Euphony Demo to recognize sound data.
 - [Euphony Speaker](https://github.com/designe/euphony-speaker) : Euphony Demo to generate sound data.
-
+- [Sound Helper](https://github.com/euphony-io/sound-helper) : Euphony dev team, Eutophia sample project
```

Comment:
I think it would be better to put a brief description of the app like your PR description rather than "Euphony dev team, Eutophia sample project". 😄

Reply:
Updated description!
Thanks for your comment